pdns/pdns_recursor.cc

```diff
@@ -3347,36 +3354,39 @@ static int serviceMain(int argc, char*argv[])
     s_avoidUdpSourcePorts.insert(port);
   }
 
+  /* This thread handles the web server, carbon, statistics and the control channel */
+  std::thread handlerThread(recursorThread, -1, false);
```

Comment:
if -1 is a magic t_id, maybe stick this into a #define or const?

Reply:
Agreed, I'll do that. I'm also pondering adding the possibility to have more than one distributor thread so this whole logic will have to be refactored anyway (but in a different PR).